app/src/main/java/chat/rocket/android/app/RocketChatApplication.kt

```diff
@@ -119,118 +115,7 @@ class RocketChatApplication : Application(), HasActivityInjector, HasServiceInje
-            val session = realm.where(RealmSession::class.java).findFirst()
-
-            migratePublicSettings(url, realm)
-            if (user != null && session != null) {
-                val authToken = session.token
-                settingsRepository.get(url)
-                migrateServerInfo(url, authToken!!, settingsRepository.get(url), user)
-            }
-            realm.close()
-        }
-        migrateCurrentServer(serversInfoList)
-        serverRealm.close()
-        localRepository.setMigrated(true)
-    }
-
-    private fun migrateServerInfo(url: String, authToken: String, settings: PublicSettings, user: RealmUser) {
-        val userId = user._id
-        val avatar = url.avatarUrl(user.username!!)
-        val icon = settings.favicon()?.let {
-            url.serverLogoUrl(it)
-        }
-        val logo = settings.wideTile()?.let {
-            url.serverLogoUrl(it)
-        }
-        val account = Account(url, icon, logo, user.username!!, avatar)
-        launch(CommonPool) {
-            tokenRepository.save(url, Token(userId!!, authToken))
-            accountRepository.save(account)
-        }
-    }
-
-    private fun migratePublicSettings(url: String, realm: Realm) {
-        val settings = realm.where(RealmPublicSetting::class.java).findAll()
-
-        val serverSettings = hashMapOf<String, Value<Any>>()
-        settings.toList().forEach { setting ->
-            val type = setting.type!!
-            val value = setting.value!!
-
-            val convertedSetting = when (type) {
-                "string" -> Value(value)
-                "language" -> Value(value)
-                "boolean" -> Value(value.toBoolean())
-                "int" -> try {
-                    Value(value.toInt())
-                } catch (ex: NumberFormatException) {
-                    Value(0)
-                }
-                else -> null // ignore
-            }
-
-            if (convertedSetting != null) {
-                val id = setting._id!!
-                serverSettings.put(id, convertedSetting)
-            }
-        }
-        settingsRepository.save(url, serverSettings)
-    }
-
-    private fun migrateCurrentServer(serversList: List<RealmBasedServerInfo>) {
-        if (getCurrentServerInteractor.get() == null) {
-            var currentServer = getSharedPreferences("cache", Context.MODE_PRIVATE)
-                    .getString("KEY_SELECTED_SERVER_HOSTNAME", null)
-
-            currentServer = if (serversList.isNotEmpty()) {
-                val server = serversList.find { it.hostname == currentServer }
-                val hostname = server!!.hostname
-                if (server.insecure) {
-                    "http://$hostname"
-                } else {
-                    "https://$hostname"
-                }
-            } else {
-                "http://$currentServer"
-            }
-            saveCurrentServerRepository.save(currentServer)
-        }
+        // TODO - remove REALM files.
```

Comment:
Looks like the REALM files were removed, so you can remove this TODO too and the commented imports

Reply:
Actually this is about the realm database files, I need to investigate the path where they are stored to clean up